buildpipeline/dotnet-standard-win.json

```diff
@@ -148,6 +220,22 @@
     "PB_GitHubAuthToken": {
       "value": null,
       "isSecret": true
+    },
+    "PB_AzureFeedUrl": {
```

Comment:
What do these URL's look like?  If they are ever authenticated URL's then you should mark them `isSecret: true`